packages/formbricks-api-wrapper/src/utils/makeRequest.ts

```diff
@@ -0,0 +1,37 @@
+import fetch from "node-fetch";
+
+import { Result, err, ok, wrapThrows } from "@formbricks/errors";
+import { NetworkError, ApiResponse } from "../types/index";
+
+export async function makeRequest<T = any, E = any>(
```

Comment:
Please take a look at the errors package and try to simplify this generic type as this is hard to understand. (If it makes sense?)

Reply:
done! please take a look now